go/test/endtoend/vreplication/helper.go

```diff
@@ -92,6 +93,22 @@ func waitForQueryToExecute(t *testing.T, conn *mysql.Conn, database string, quer
 	}
 }
 
+func verifyNoInternalTables(t *testing.T, conn *mysql.Conn, database string) {
+	t.Logf("Checking that no internal tables exist in keyspace: %s", database)
+	qr := execVtgateQuery(t, conn, database, "show tables")
+	require.NotNil(t, qr)
+	require.NotNil(t, qr.Rows)
+	found := false
+	for _, row := range qr.Rows {
```

Comment:
To gain more visibility into failure, I'd suggest:
```go
   tableName := row[0].ToString()
   assert.False(t, schema.IsInternalOperationTableName(tableName), "found internal table %s", tableName)
```

then, remove the block with `t.Logf` and the `require...` that follows.

Reply:
Good call. I was being noisy during testing to see what was happening, but don't need that in ever CI run going forward.